src/packagedcode/npm.py

```diff
@@ -168,7 +168,12 @@ def walk_npm(cls, resource, codebase, depth=0):
                     yield subchild
 
 
-def get_urls(namespace, name, version, **kwargs):
+def get_urls(namespace, name, version, is_upm=False, **kwargs):
+    if is_upm:
+        return dict(
+            repository_homepage_url='https://docs.unity3d.com/Manual/Packages.html',
```

Comment:
Is there some sorts home page for each package or package version? if not, this is best left empty